Content.Server/GameObjects/Components/Research/ResearchConsoleComponent.cs

```diff
@@ -30,13 +32,19 @@ public class ResearchConsoleComponent : SharedResearchConsoleComponent, IActivat
 
         private BoundUserInterface _userInterface;
         private ResearchClientComponent _client;
+        private PowerDeviceComponent _powerDevice;
+        private AppearanceComponent _appearance;
```

Comment:
You don't appear to be using this field anywhere?

Reply:
Oops, leftover.